materialize-sqlserver/sqlgen.go

```diff
@@ -188,11 +206,13 @@ END;
 -- Server does not support modifying multiple columns in a single statement.
 
 {{ define "alterTableColumns" }}
+{{- if $.AddColumns }}
```

Comment:
Is this conditional needed? We shouldn't ever be rendering this template unless there are columns to add.